lib/supavisor/db_handler.ex

```diff
@@ -82,7 +84,8 @@ defmodule Supavisor.DbHandler do
     {:keep_state_and_data, {:next_event, :internal, :connect}}
   end
 
-  def handle_event(:info, {:tcp, _, bin}, :authentication, data) do
+  def handle_event(:info, {tcp_or_ssl, _, bin}, :authentication, data)
+      when tcp_or_ssl in [:tcp, :ssl] do
```

Comment:
Another way to do this: have a `handle_event` for each of `:tcp` and `:ssl` and route to the same place.